apps/i18n/common/en_us.json

```diff
@@ -841,11 +841,13 @@
   "levelType": "Level Type",
   "libraryClassImport": "Import library from my class",
   "libraryCodeError": "We can't publish your library because there is an error in the code. Go look for the square red error indicator and fix the errors.",
+  "libraryCreatorError": "There was an error creating your library. Contact [support@code.org](mailto:support@code.org) to resolve the issue.",
```

Comment:
Oops. No that doesn't work and shouldn't (I think it _can_ work if you use one of the `dangerouslySet...` functions). I don't think even really need the link here (it seems like we give this a link only sometimes) since this would be an odd case to hit. Pulling that out.